app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def dashboard_cards_by_role
+    {
+      challenge_manager: [
+        { image_path: 'folder_open', href: 'manage_submissions', alt: 'submissions', title: 'Submissions',
+          subtitle: 'Manage submissions, evaluations, and evaluators.' },
+        { image_path: 'content_copy', href: 'evaluation_forms', alt: 'evaluation forms', title: 'Evaluation Forms',
+          subtitle: 'Create and manage evaluation forms.' },
+        { image_path: 'map', href: 'user_guide', alt: 'user guides', title: 'User Guides',
+          subtitle: 'Learn how to make the most of the platform.' },
+        { image_path: 'support_agent', href: 'federal-agency-faqs', alt: 'help', title: 'Help',
+          subtitle: 'Get support on the Challenge.Gov platform.' }
+      ],
+      solver: [],
+      evaluator: []
+    }
+  end
+
+  def dashboard_title
+    {
+      challenge_manager: "Challenge Manager Evaluation Dashboard"
```

Comment:
Figma just shows "Dashboard" for the title of both challenge_manager and evaluator roles, but I think it's fine to keep it dynamic rather than just hard-coding the page title for now.

Reply:
There's different titles and card content depending on where you look in Figma. I'll use what's pasted here in this PR as source of truth for now.